src/Microsoft.AspNetCore.Razor.Runtime/TagHelpers/TagHelperContent.cs

```diff
@@ -19,14 +19,9 @@ public abstract class TagHelperContent : IHtmlContentBuilder
         public abstract bool IsModified { get; }
 
         /// <summary>
-        /// Gets a value indicating whether the content is empty.
+        /// Gets a value indicating whether the content is empty or whitespace.
         /// </summary>
-        public abstract bool IsEmpty { get; }
-
-        /// <summary>
-        /// Gets a value indicating whether the content is whitespace.
-        /// </summary>
-        public abstract bool IsWhiteSpace { get; }
+        public abstract bool IsEmptyOrWhiteSpace { get; }
```

Comment:
@rynowak and I talked about this guy. It's a feature that we've never used or seen used and more importantly can be easily replaced. Didn't feel that this was important enough to warrant a larger conversation/decision with more people.